### PR TITLE
OSDOCS#8969: Adding descheduler 5.0.0 release notes

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.15

Issue:
https://issues.redhat.com/browse/OSDOCS-8969

Link to docs preview:
https://70785--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/scheduling/descheduler/nodes-descheduler-release-notes#descheduler-operator-release-notes-5.0.0

QE review:
- [x] QE has approved this change.

Additional information:

**Not to be merged until 6 March. The advisory link won't work until this day.**

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@kasturinarra @ingvagabund Here is the new PR for just the 5.0.0 release notes. PTAL when you get a chance, thanks!

Preview: https://70785--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/scheduling/descheduler/nodes-descheduler-release-notes#descheduler-operator-release-notes-5.0.0

**Comment by kasturinarra** [comment]
@bergerhoffer content looks good to me, but i see some formatting is not right, will this be corrected when the doc is published, below is how i see it now, not sure if it is just me or you also see it the same way.

Screenshot from 2024-01-25 11-48-46

Especially `New features and Enhancements` , `Bug Fixes`.

File: nodes/scheduling/descheduler/nodes-descheduler-release-notes.adoc [hunk 1]
```diff
@@ -10,7 +10,27 @@ The {descheduler-operator} allows you to evict pods so that they can be reschedu
...
+
+[id="descheduler-operator-release-notes-5.0.0"]
+== Release notes for {descheduler-operator} 5.0.0
+
+Issued: 2024-02-28
+
+The following advisory is available for the {descheduler-operator} 5.0.0:
+
+* link:https://access.redhat.com/errata/RHEA-2024:0302[RHEA-2024:0302]
+[id="descheduler-operator-5.0.0-new-features-and-enhancements"]
```
Comment (agantony) [inline nodes/scheduling/descheduler/nodes-descheduler-release-notes.adoc:23]:
```suggestion

[id="descheduler-operator-5.0.0-new-features-and-enhancements"]
```

File: nodes/scheduling/descheduler/nodes-descheduler-release-notes.adoc [hunk 2]
```diff
@@ -10,7 +10,27 @@ The {descheduler-operator} allows you to evict pods so that they can be reschedu
...
+=== New features and enhancements
+
+* With this release, the {descheduler-operator} releases updates independently from the {product-title} minor version release stream.
+[id="descheduler-operator-5.0.0-bug-fixes"]
```
Comment (agantony) [inline nodes/scheduling/descheduler/nodes-descheduler-release-notes.adoc:27]:
```suggestion

[id="descheduler-operator-5.0.0-bug-fixes"]
```

File: nodes/scheduling/descheduler/nodes-descheduler-release-notes.adoc [hunk 3]
```diff
@@ -10,7 +10,27 @@ The {descheduler-operator} allows you to evict pods so that they can be reschedu
...
+=== Bug fixes
+
+* Previously, the descheduler pod logs showed the following warning about the Operator's version: `failed to convert Descheduler minor version to float`. With this update, the warning is no longer shown. (link:https://issues.redhat.com/browse/OCPBUGS-14042[*OCPBUGS-14042*])
```
Comment (agantony) [inline nodes/scheduling/descheduler/nodes-descheduler-release-notes.adoc:30]:
```suggestion
* Previously, the descheduler pod logs showed the following warning about the Operator's version: 

[source,text]
----
`failed to convert Descheduler minor version to float`. 
----

With this update, the warning is no longer shown. (link:https://issues.redhat.com/browse/OCPBUGS-14042[*OCPBUGS-14042*])
```

File: nodes/scheduling/descheduler/nodes-descheduler-release-notes.adoc [hunk 4]
```diff
@@ -10,7 +10,27 @@ The {descheduler-operator} allows you to evict pods so that they can be reschedu
...
+
+Issued: 2024-02-28
+
+The following advisory is available for the {descheduler-operator} 5.0.0:
+
+* link:https://access.redhat.com/errata/RHEA-2024:0302[RHEA-2024:0302]
+[id="descheduler-operator-5.0.0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* With this release, the {descheduler-operator} releases updates independently from the {product-title} minor version release stream.
```
Comment (agantony) [inline nodes/scheduling/descheduler/nodes-descheduler-release-notes.adoc:26]:
```suggestion
* With this release, the {descheduler-operator} releases updates independent of the {product-title} minor version release stream.
```

**Review by agantony — COMMENTED** [review]
Added a few suggestions for your consideration; overall lgtm!

**Comment by bergerhoffer** [comment]
Sorry all, looks like I forgot to check the preview before asking for reviews, yikes :grimacing: 

Updated and ready for re-review @kasturinarra

**Comment by ingvagabund** [comment]
Looks good. We are still discussing the subs type change (to be decided whether we can do it). Though, that can be announced later.

**Comment by kasturinarra** [comment]
> Looks good. We are still discussing the subs type change (to be decided whether we can do it). Though, that can be announced later.

@ingvagabund what is the subs type change that you are referring ? Also may i know where is this discussion happening ?

**Comment by ingvagabund** [comment]
KDO's subscription type is currently set to "OKE and up". I'd like to change it to "OCP and up" since we have actually never supported KDO on OKD.

**Comment by ingvagabund** [comment]
There are multiple places where this is getting discussed. Some are private, some are public. E.g. https://chat.google.com/room/AAAAZrx3KlI/pmqdq9IPzow

**Comment by kasturinarra** [comment]
> KDO's subscription type is currently set to "OKE and up". I'd like to change it to "OCP and up" since we have actually never supported KDO on OKD.

Thanks jan, i completely forgot about this. Yea  and aware of the discussion channels, thanks !!

**Comment by ingvagabund** [comment]
No worries. There are already too many discussions in parallel for too many topics.

**Comment by bergerhoffer** [comment]
@kasturinarra Can you take a last look and see if this looks good now?

**Comment by bergerhoffer** [comment]
@kasturinarra can you please re-review this with the addition of the descheduler not being supported on OKE anymore?

https://70785--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/scheduling/descheduler/nodes-descheduler-release-notes#descheduler-operator-release-notes-5.0.0

**Comment by bergerhoffer** [comment]
Updated to reflect new GA date of 12 March

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.16` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.15. And any PR going into `main` must also target the latest version branch (enterprise-4.16).

If the update in your PR does NOT apply to version 4.16 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main

**Pull request merged**